<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSearch</class>
 <widget class="QDialog" name="DialogSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>672</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>321</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>321</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Search Bible</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>125</y>
     <width>321</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>9</y>
     <width>306</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enter Search Text here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSearch">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>35</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Restart Search From Beginning</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../CWP/pictureResources.qrc">
     <normaloff>:/icons/Pics/search_ideogram.png</normaloff>:/icons/Pics/search_ideogram.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSettings">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>35</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Search Settings</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../CWP/pictureResources.qrc">
     <normaloff>:/icons/Pics/smallgear.png</normaloff>:/icons/Pics/smallgear.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClear">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>35</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>search more</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../CWP/pictureResources.qrc">
     <normaloff>:/icons/Pics/sweep.png</normaloff>:/icons/Pics/sweep.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelFound">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>78</y>
     <width>296</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>67</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelinBooks">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>296</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClose_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>635</y>
     <width>233</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>45</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../CWP/pictureResources.qrc"/>
 </resources>
 <connections/>
</ui>
